[opentelemetry-cpp] Build failure

Package: opentelemetry-cpp[core,otlp,otlp-grpc,otlp-http]:x64-uwp -> 1.12.0

**Host Environment**

- Host: x64-windows
- Compiler: MSVC 19.37.32822.0
-    vcpkg-tool version: 2999-12-31-unknownhash
    vcpkg-scripts version: ad741ffa2 2023-11-01 (3 minutes ago)

**To Reproduce**

`vcpkg install --enforce-port-checks`
**Failure logs**

```
-- Note: opentelemetry-cpp only supports static library linkage. Building static library.
-- Using cached open-telemetry-opentelemetry-cpp-v1.12.0.tar.gz.
-- Extracting source C:/v/downloads/open-telemetry-opentelemetry-cpp-v1.12.0.tar.gz
-- Applying patch add-missing-dependencies.patch
-- Applying patch add-missing-find-dependency.patch
-- Using source at C:/v/vcpkg3/buildtrees/opentelemetry-cpp/src/v1.12.0-c900296c5a.clean
-- Found external ninja('1.11.0').
-- Configuring x64-uwp
-- Building x64-uwp-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
    Command failed: C:/v/downloads/tools/cmake-3.27.1-windows/cmake-3.27.1-windows-i386/bin/cmake.exe --build . --config Debug --target install -- -v -j31
    Working Directory: C:/v/vcpkg3/buildtrees/opentelemetry-cpp/x64-uwp-dbg
    See logs for more information:
      C:\v\vcpkg3\buildtrees\opentelemetry-cpp\install-x64-uwp-dbg-out.log

Call Stack (most recent call first):
  vcpkg_installed/x64-windows/share/vcpkg-cmake/vcpkg_cmake_build.cmake:74 (vcpkg_execute_build_process)
  vcpkg_installed/x64-windows/share/vcpkg-cmake/vcpkg_cmake_install.cmake:16 (vcpkg_cmake_build)
  ports/opentelemetry-cpp/portfile.cmake:64 (vcpkg_cmake_install)
  scripts/ports.cmake:168 (include)



```
<details><summary>C:\v\vcpkg3\buildtrees\opentelemetry-cpp\install-x64-uwp-dbg-out.log</summary>

```
Change Dir: 'C:/v/vcpkg3/buildtrees/opentelemetry-cpp/x64-uwp-dbg'

Run Build Command(s): "C:/Program Files/Microsoft Visual Studio/2022/Community/Common7/IDE/CommonExtensions/Microsoft/CMake/Ninja/ninja.exe" -v -v -j31 install
[1/147] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\x64\cl.exe   /TP -DABSL_CONSUME_DLL -DHAVE_ABSEIL -DNOMINMAX -DOPENTELEMETRY_ABI_VERSION_NO=1 -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\x64-uwp-dbg\generated\third_party\opentelemetry-proto -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\api\include -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\sdk\include -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\sdk -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\ext\include -external:IC:\v\vcpkg3\vcpkg_installed\x64-uwp\include -external:W0 /DWIN32 /D_WINDOWS /D_UNICODE /DUNICODE /DWINAPI_FAMILY=WINAPI_FAMILY_APP /D__WRL_NO_DEFAULT_LIB__ /nologo /Z7 /MP /GS /Gd /Gm- /W3 /WX- /Zc:wchar_t /Zc:inline /Zc:forScope /fp:precise /Oy- /EHsc  /utf-8  /Zc:__cplusplus /D_DEBUG /MDd /Od /RTC1 /showIncludes /Fosdk\src\common\CMakeFiles\opentelemetry_common.dir\platform\fork_windows.cc.obj /Fdsdk\src\common\CMakeFiles\opentelemetry_common.dir\opentelemetry_common.pdb /FS -c C:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\sdk\src\common\platform\fork_windows.cc
[2/147] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\x64\cl.exe   /TP -DABSL_CONSUME_DLL -DHAVE_ABSEIL -DNOMINMAX -DOPENTELEMETRY_ABI_VERSION_NO=1 -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\x64-uwp-dbg\generated\third_party\opentelemetry-proto -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\api\include -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\sdk\include -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\sdk -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\ext\include -external:IC:\v\vcpkg3\vcpkg_installed\x64-uwp\include -external:W0 /DWIN32 /D_WINDOWS /D_UNICODE /DUNICODE /DWINAPI_FAMILY=WINAPI_FAMILY_APP /D__WRL_NO_DEFAULT_LIB__ /nologo /Z7 /MP /GS /Gd /Gm- /W3 /WX- /Zc:wchar_t /Zc:inline /Zc:forScope /fp:precise /Oy- /EHsc  /utf-8  /Zc:__cplusplus /D_DEBUG /MDd /Od /RTC1 /showIncludes /Fosdk\src\trace\CMakeFiles\opentelemetry_trace.dir\exporter.cc.obj /Fdsdk\src\trace\CMakeFiles\opentelemetry_trace.dir\opentelemetry_trace.pdb /FS -c C:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\sdk\src\trace\exporter.cc
[3/147] cmd.exe /C "cd /D C:\v\vcpkg3\buildtrees\opentelemetry-cpp\x64-uwp-dbg && C:\v\downloads\tools\cmake-3.27.1-windows\cmake-3.27.1-windows-i386\bin\cmake.exe -Dcfgdir= -P C:/v/vcpkg3/buildtrees/opentelemetry-cpp/x64-uwp-dbg/opentelemetry-proto-prefix/tmp/opentelemetry-proto-mkdirs.cmake && C:\v\downloads\tools\cmake-3.27.1-windows\cmake-3.27.1-windows-i386\bin\cmake.exe -E touch C:/v/vcpkg3/buildtrees/opentelemetry-cpp/x64-uwp-dbg/opentelemetry-proto-prefix/src/opentelemetry-proto-stamp/opentelemetry-proto-mkdir"
...
Skipped 63 lines
...
[67/147] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\x64\cl.exe   /TP -DABSL_CONSUME_DLL -DHAVE_ABSEIL -DNOMINMAX -DOPENTELEMETRY_ABI_VERSION_NO=1 -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\x64-uwp-dbg\generated\third_party\opentelemetry-proto -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\api\include -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\sdk\include -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\sdk -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\ext\include -external:IC:\v\vcpkg3\vcpkg_installed\x64-uwp\include -external:W0 /DWIN32 /D_WINDOWS /D_UNICODE /DUNICODE /DWINAPI_FAMILY=WINAPI_FAMILY_APP /D__WRL_NO_DEFAULT_LIB__ /nologo /Z7 /MP /GS /Gd /Gm- /W3 /WX- /Zc:wchar_t /Zc:inline /Zc:forScope /fp:precise /Oy- /EHsc  /utf-8  /Zc:__cplusplus /D_DEBUG /MDd /Od /RTC1 /showIncludes /Fosdk\src\logs\CMakeFiles\opentelemetry_logs.dir\event_logger.cc.obj /Fdsdk\src\logs\CMakeFiles\opentelemetry_logs.dir\opentelemetry_logs.pdb /FS -c C:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\sdk\src\logs\event_logger.cc
[68/147] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\x64\cl.exe   /TP -DABSL_CONSUME_DLL -DHAVE_ABSEIL -DNOMINMAX -DOPENTELEMETRY_ABI_VERSION_NO=1 -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\x64-uwp-dbg\generated\third_party\opentelemetry-proto -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\api\include -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\sdk\include -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\sdk -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\ext\include -external:IC:\v\vcpkg3\vcpkg_installed\x64-uwp\include -external:W0 /DWIN32 /D_WINDOWS /D_UNICODE /DUNICODE /DWINAPI_FAMILY=WINAPI_FAMILY_APP /D__WRL_NO_DEFAULT_LIB__ /nologo /Z7 /MP /GS /Gd /Gm- /W3 /WX- /Zc:wchar_t /Zc:inline /Zc:forScope /fp:precise /Oy- /EHsc  /utf-8  /Zc:__cplusplus /D_DEBUG /MDd /Od /RTC1 /showIncludes /Fosdk\src\resource\CMakeFiles\opentelemetry_resources.dir\resource_detector.cc.obj /Fdsdk\src\resource\CMakeFiles\opentelemetry_resources.dir\opentelemetry_resources.pdb /FS -c C:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\sdk\src\resource\resource_detector.cc
[69/147] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\x64\cl.exe   /TP -DABSL_CONSUME_DLL -DHAVE_ABSEIL -DNOMINMAX -DOPENTELEMETRY_ABI_VERSION_NO=1 -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\x64-uwp-dbg\generated\third_party\opentelemetry-proto -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\api\include -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\sdk\include -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\sdk -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\ext\include -external:IC:\v\vcpkg3\vcpkg_installed\x64-uwp\include -external:W0 /DWIN32 /D_WINDOWS /D_UNICODE /DUNICODE /DWINAPI_FAMILY=WINAPI_FAMILY_APP /D__WRL_NO_DEFAULT_LIB__ /nologo /Z7 /MP /GS /Gd /Gm- /W3 /WX- /Zc:wchar_t /Zc:inline /Zc:forScope /fp:precise /Oy- /EHsc  /utf-8  /Zc:__cplusplus /D_DEBUG /MDd /Od /RTC1 /showIncludes /Fosdk\src\metrics\CMakeFiles\opentelemetry_metrics.dir\meter_provider_factory.cc.obj /Fdsdk\src\metrics\CMakeFiles\opentelemetry_metrics.dir\opentelemetry_metrics.pdb /FS -c C:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\sdk\src\metrics\meter_provider_factory.cc
[70/147] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\x64\cl.exe   /TP -DABSL_CONSUME_DLL -DHAVE_ABSEIL -DNOMINMAX -DOPENTELEMETRY_ABI_VERSION_NO=1 -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\x64-uwp-dbg\generated\third_party\opentelemetry-proto -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\api\include -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\sdk\include -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\sdk -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\ext\include -external:IC:\v\vcpkg3\vcpkg_installed\x64-uwp\include -external:W0 /DWIN32 /D_WINDOWS /D_UNICODE /DUNICODE /DWINAPI_FAMILY=WINAPI_FAMILY_APP /D__WRL_NO_DEFAULT_LIB__ /nologo /Z7 /MP /GS /Gd /Gm- /W3 /WX- /Zc:wchar_t /Zc:inline /Zc:forScope /fp:precise /Oy- /EHsc  /utf-8  /Zc:__cplusplus /D_DEBUG /MDd /Od /RTC1 /showIncludes /Foext\src\http\client\curl\CMakeFiles\opentelemetry_http_client_curl.dir\http_operation_curl.cc.obj /Fdext\src\http\client\curl\CMakeFiles\opentelemetry_http_client_curl.dir\opentelemetry_http_client_curl.pdb /FS -c C:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\ext\src\http\client\curl\http_operation_curl.cc
C:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\ext\src\http\client\curl\http_operation_curl.cc(890): warning C4244: 'argument': conversion from '_Rep' to 'long', possible loss of data
        with
        [
            _Rep=__int64
        ]
[71/147] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\x64\cl.exe   /TP -DABSL_CONSUME_DLL -DHAVE_ABSEIL -DNOMINMAX -DOPENTELEMETRY_ABI_VERSION_NO=1 -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\x64-uwp-dbg\generated\third_party\opentelemetry-proto -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\api\include -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\sdk\include -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\sdk -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\ext\include -external:IC:\v\vcpkg3\vcpkg_installed\x64-uwp\include -external:W0 /DWIN32 /D_WINDOWS /D_UNICODE /DUNICODE /DWINAPI_FAMILY=WINAPI_FAMILY_APP /D__WRL_NO_DEFAULT_LIB__ /nologo /Z7 /MP /GS /Gd /Gm- /W3 /WX- /Zc:wchar_t /Zc:inline /Zc:forScope /fp:precise /Oy- /EHsc  /utf-8  /Zc:__cplusplus /D_DEBUG /MDd /Od /RTC1 /showIncludes /Fosdk\src\metrics\CMakeFiles\opentelemetry_metrics.dir\sync_instruments.cc.obj /Fdsdk\src\metrics\CMakeFiles\opentelemetry_metrics.dir\opentelemetry_metrics.pdb /FS -c C:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\sdk\src\metrics\sync_instruments.cc
[72/147] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\x64\cl.exe   /TP -DABSL_CONSUME_DLL -DHAVE_ABSEIL -DNOMINMAX -DOPENTELEMETRY_ABI_VERSION_NO=1 -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\x64-uwp-dbg\generated\third_party\opentelemetry-proto -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\api\include -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\sdk\include -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\sdk -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\ext\include -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\exporters\ostream\include -external:IC:\v\vcpkg3\vcpkg_installed\x64-uwp\include -external:W0 /DWIN32 /D_WINDOWS /D_UNICODE /DUNICODE /DWINAPI_FAMILY=WINAPI_FAMILY_APP /D__WRL_NO_DEFAULT_LIB__ /nologo /Z7 /MP /GS /Gd /Gm- /W3 /WX- /Zc:wchar_t /Zc:inline /Zc:forScope /fp:precise /Oy- /EHsc  /utf-8  /Zc:__cplusplus /D_DEBUG /MDd /Od /RTC1 /showIncludes /Foexporters\ostream\CMakeFiles\opentelemetry_exporter_ostream_logs.dir\src\log_record_exporter_factory.cc.obj /Fdexporters\ostream\CMakeFiles\opentelemetry_exporter_ostream_logs.dir\opentelemetry_exporter_ostream_logs.pdb /FS -c C:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\exporters\ostream\src\log_record_exporter_factory.cc
[73/147] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\x64\cl.exe   /TP -DABSL_CONSUME_DLL -DHAVE_ABSEIL -DNOMINMAX -DOPENTELEMETRY_ABI_VERSION_NO=1 -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\x64-uwp-dbg\generated\third_party\opentelemetry-proto -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\api\include -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\sdk\include -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\sdk -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\ext\include -external:IC:\v\vcpkg3\vcpkg_installed\x64-uwp\include -external:W0 /DWIN32 /D_WINDOWS /D_UNICODE /DUNICODE /DWINAPI_FAMILY=WINAPI_FAMILY_APP /D__WRL_NO_DEFAULT_LIB__ /nologo /Z7 /MP /GS /Gd /Gm- /W3 /WX- /Zc:wchar_t /Zc:inline /Zc:forScope /fp:precise /Oy- /EHsc  /utf-8  /Zc:__cplusplus /D_DEBUG /MDd /Od /RTC1 /showIncludes /Fosdk\src\resource\CMakeFiles\opentelemetry_resources.dir\resource.cc.obj /Fdsdk\src\resource\CMakeFiles\opentelemetry_resources.dir\opentelemetry_resources.pdb /FS -c C:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\sdk\src\resource\resource.cc
[74/147] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\x64\cl.exe   /TP -DABSL_CONSUME_DLL -DHAVE_ABSEIL -DNOMINMAX -DOPENTELEMETRY_ABI_VERSION_NO=1 -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\x64-uwp-dbg\generated\third_party\opentelemetry-proto -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\api\include -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\sdk\include -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\sdk -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\ext\include -external:IC:\v\vcpkg3\vcpkg_installed\x64-uwp\include -external:W0 /DWIN32 /D_WINDOWS /D_UNICODE /DUNICODE /DWINAPI_FAMILY=WINAPI_FAMILY_APP /D__WRL_NO_DEFAULT_LIB__ /nologo /Z7 /MP /GS /Gd /Gm- /W3 /WX- /Zc:wchar_t /Zc:inline /Zc:forScope /fp:precise /Oy- /EHsc  /utf-8  /Zc:__cplusplus /D_DEBUG /MDd /Od /RTC1 /showIncludes /Fosdk\src\metrics\CMakeFiles\opentelemetry_metrics.dir\meter.cc.obj /Fdsdk\src\metrics\CMakeFiles\opentelemetry_metrics.dir\opentelemetry_metrics.pdb /FS -c C:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\sdk\src\metrics\meter.cc
[75/147] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\x64\cl.exe   /TP -DABSL_CONSUME_DLL -DHAVE_ABSEIL -DNOMINMAX -DOPENTELEMETRY_ABI_VERSION_NO=1 -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\x64-uwp-dbg\generated\third_party\opentelemetry-proto -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\api\include -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\sdk\include -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\sdk -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\ext\include -external:IC:\v\vcpkg3\vcpkg_installed\x64-uwp\include -external:W0 /DWIN32 /D_WINDOWS /D_UNICODE /DUNICODE /DWINAPI_FAMILY=WINAPI_FAMILY_APP /D__WRL_NO_DEFAULT_LIB__ /nologo /Z7 /MP /GS /Gd /Gm- /W3 /WX- /Zc:wchar_t /Zc:inline /Zc:forScope /fp:precise /Oy- /EHsc  /utf-8  /Zc:__cplusplus /D_DEBUG /MDd /Od /RTC1 /showIncludes /Fosdk\src\logs\CMakeFiles\opentelemetry_logs.dir\logger_context_factory.cc.obj /Fdsdk\src\logs\CMakeFiles\opentelemetry_logs.dir\opentelemetry_logs.pdb /FS -c C:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\sdk\src\logs\logger_context_factory.cc
[76/147] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\x64\cl.exe   /TP -DABSL_CONSUME_DLL -DHAVE_ABSEIL -DNOMINMAX -DOPENTELEMETRY_ABI_VERSION_NO=1 -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\x64-uwp-dbg\generated\third_party\opentelemetry-proto -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\api\include -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\sdk\include -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\sdk -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\ext\include -external:IC:\v\vcpkg3\vcpkg_installed\x64-uwp\include -external:W0 /DWIN32 /D_WINDOWS /D_UNICODE /DUNICODE /DWINAPI_FAMILY=WINAPI_FAMILY_APP /D__WRL_NO_DEFAULT_LIB__ /nologo /Z7 /MP /GS /Gd /Gm- /W3 /WX- /Zc:wchar_t /Zc:inline /Zc:forScope /fp:precise /Oy- /EHsc  /utf-8  /Zc:__cplusplus /D_DEBUG /MDd /Od /RTC1 /showIncludes /Foext\src\http\client\curl\CMakeFiles\opentelemetry_http_client_curl.dir\http_client_curl.cc.obj /Fdext\src\http\client\curl\CMakeFiles\opentelemetry_http_client_curl.dir\opentelemetry_http_client_curl.pdb /FS -c C:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\ext\src\http\client\curl\http_client_curl.cc
[77/147] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\x64\cl.exe   /TP -DABSL_CONSUME_DLL -DHAVE_ABSEIL -DNOMINMAX -DOPENTELEMETRY_ABI_VERSION_NO=1 -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\x64-uwp-dbg\generated\third_party\opentelemetry-proto -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\api\include -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\sdk\include -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\sdk -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\ext\include -external:IC:\v\vcpkg3\vcpkg_installed\x64-uwp\include -external:W0 /DWIN32 /D_WINDOWS /D_UNICODE /DUNICODE /DWINAPI_FAMILY=WINAPI_FAMILY_APP /D__WRL_NO_DEFAULT_LIB__ /nologo /Z7 /MP /GS /Gd /Gm- /W3 /WX- /Zc:wchar_t /Zc:inline /Zc:forScope /fp:precise /Oy- /EHsc  /utf-8  /Zc:__cplusplus /D_DEBUG /MDd /Od /RTC1 /showIncludes /Fosdk\src\logs\CMakeFiles\opentelemetry_logs.dir\multi_recordable.cc.obj /Fdsdk\src\logs\CMakeFiles\opentelemetry_logs.dir\opentelemetry_logs.pdb /FS -c C:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\sdk\src\logs\multi_recordable.cc
[78/147] cmd.exe /C "cd . && C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\x64\lib.exe  /machine:x64 /nologo /out:sdk\src\resource\opentelemetry_resources.lib sdk\src\resource\CMakeFiles\opentelemetry_resources.dir\resource.cc.obj sdk\src\resource\CMakeFiles\opentelemetry_resources.dir\resource_detector.cc.obj  && cd ."
[79/147] cmd.exe /C "cd . && C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\x64\lib.exe  /machine:x64 /nologo /out:ext\src\http\client\curl\opentelemetry_http_client_curl.lib ext\src\http\client\curl\CMakeFiles\opentelemetry_http_client_curl.dir\http_client_factory_curl.cc.obj ext\src\http\client\curl\CMakeFiles\opentelemetry_http_client_curl.dir\http_client_curl.cc.obj ext\src\http\client\curl\CMakeFiles\opentelemetry_http_client_curl.dir\http_operation_curl.cc.obj  && cd ."
[80/147] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\x64\cl.exe   /TP -DABSL_CONSUME_DLL -DHAVE_ABSEIL -DNOMINMAX -DOPENTELEMETRY_ABI_VERSION_NO=1 -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\x64-uwp-dbg\generated\third_party\opentelemetry-proto -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\api\include -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\sdk\include -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\sdk -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\ext\include -external:IC:\v\vcpkg3\vcpkg_installed\x64-uwp\include -external:W0 /DWIN32 /D_WINDOWS /D_UNICODE /DUNICODE /DWINAPI_FAMILY=WINAPI_FAMILY_APP /D__WRL_NO_DEFAULT_LIB__ /nologo /Z7 /MP /GS /Gd /Gm- /W3 /WX- /Zc:wchar_t /Zc:inline /Zc:forScope /fp:precise /Oy- /EHsc  /utf-8  /Zc:__cplusplus /D_DEBUG /MDd /Od /RTC1 /showIncludes /Fosdk\src\logs\CMakeFiles\opentelemetry_logs.dir\readable_log_record.cc.obj /Fdsdk\src\logs\CMakeFiles\opentelemetry_logs.dir\opentelemetry_logs.pdb /FS -c C:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\sdk\src\logs\readable_log_record.cc
[81/147] cmd.exe /C "cd . && C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\x64\lib.exe  /machine:x64 /nologo /out:sdk\src\trace\opentelemetry_trace.lib sdk\src\trace\CMakeFiles\opentelemetry_trace.dir\tracer_context.cc.obj sdk\src\trace\CMakeFiles\opentelemetry_trace.dir\tracer_context_factory.cc.obj sdk\src\trace\CMakeFiles\opentelemetry_trace.dir\tracer_provider.cc.obj sdk\src\trace\CMakeFiles\opentelemetry_trace.dir\tracer_provider_factory.cc.obj sdk\src\trace\CMakeFiles\opentelemetry_trace.dir\tracer.cc.obj sdk\src\trace\CMakeFiles\opentelemetry_trace.dir\span.cc.obj sdk\src\trace\CMakeFiles\opentelemetry_trace.dir\exporter.cc.obj sdk\src\trace\CMakeFiles\opentelemetry_trace.dir\batch_span_processor.cc.obj sdk\src\trace\CMakeFiles\opentelemetry_trace.dir\batch_span_processor_factory.cc.obj sdk\src\trace\CMakeFiles\opentelemetry_trace.dir\simple_processor_factory.cc.obj sdk\src\trace\CMakeFiles\opentelemetry_trace.dir\samplers\always_on_factory.cc.obj sdk\src\trace\CMakeFiles\opentelemetry_trace.dir\samplers\always_off_factory.cc.obj sdk\src\trace\CMakeFiles\opentelemetry_trace.dir\samplers\parent.cc.obj sdk\src\trace\CMakeFiles\opentelemetry_trace.dir\samplers\parent_factory.cc.obj sdk\src\trace\CMakeFiles\opentelemetry_trace.dir\samplers\trace_id_ratio.cc.obj sdk\src\trace\CMakeFiles\opentelemetry_trace.dir\samplers\trace_id_ratio_factory.cc.obj sdk\src\trace\CMakeFiles\opentelemetry_trace.dir\random_id_generator.cc.obj sdk\src\trace\CMakeFiles\opentelemetry_trace.dir\random_id_generator_factory.cc.obj  && cd ."
[82/147] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\x64\cl.exe   /TP -DABSL_CONSUME_DLL -DHAVE_ABSEIL -DNOMINMAX -DOPENTELEMETRY_ABI_VERSION_NO=1 -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\x64-uwp-dbg\generated\third_party\opentelemetry-proto -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\api\include -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\sdk\include -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\sdk -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\ext\include -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\exporters\ostream\include -external:IC:\v\vcpkg3\vcpkg_installed\x64-uwp\include -external:W0 /DWIN32 /D_WINDOWS /D_UNICODE /DUNICODE /DWINAPI_FAMILY=WINAPI_FAMILY_APP /D__WRL_NO_DEFAULT_LIB__ /nologo /Z7 /MP /GS /Gd /Gm- /W3 /WX- /Zc:wchar_t /Zc:inline /Zc:forScope /fp:precise /Oy- /EHsc  /utf-8  /Zc:__cplusplus /D_DEBUG /MDd /Od /RTC1 /showIncludes /Foexporters\ostream\CMakeFiles\opentelemetry_exporter_ostream_span.dir\src\span_exporter_factory.cc.obj /Fdexporters\ostream\CMakeFiles\opentelemetry_exporter_ostream_span.dir\opentelemetry_exporter_ostream_span.pdb /FS -c C:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\exporters\ostream\src\span_exporter_factory.cc
[83/147] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\x64\cl.exe   /TP -DABSL_CONSUME_DLL -DHAVE_ABSEIL -DNOMINMAX -DOPENTELEMETRY_ABI_VERSION_NO=1 -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\x64-uwp-dbg\generated\third_party\opentelemetry-proto -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\api\include -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\sdk\include -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\sdk -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\ext\include -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\exporters\ostream\include -external:IC:\v\vcpkg3\vcpkg_installed\x64-uwp\include -external:W0 /DWIN32 /D_WINDOWS /D_UNICODE /DUNICODE /DWINAPI_FAMILY=WINAPI_FAMILY_APP /D__WRL_NO_DEFAULT_LIB__ /nologo /Z7 /MP /GS /Gd /Gm- /W3 /WX- /Zc:wchar_t /Zc:inline /Zc:forScope /fp:precise /Oy- /EHsc  /utf-8  /Zc:__cplusplus /D_DEBUG /MDd /Od /RTC1 /showIncludes /Foexporters\ostream\CMakeFiles\opentelemetry_exporter_ostream_metrics.dir\src\metric_exporter_factory.cc.obj /Fdexporters\ostream\CMakeFiles\opentelemetry_exporter_ostream_metrics.dir\opentelemetry_exporter_ostream_metrics.pdb /FS -c C:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\exporters\ostream\src\metric_exporter_factory.cc
[84/147] cmd.exe /C "cd . && C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\x64\lib.exe  /machine:x64 /nologo /out:sdk\src\logs\opentelemetry_logs.lib sdk\src\logs\CMakeFiles\opentelemetry_logs.dir\logger_provider.cc.obj sdk\src\logs\CMakeFiles\opentelemetry_logs.dir\logger_provider_factory.cc.obj sdk\src\logs\CMakeFiles\opentelemetry_logs.dir\logger.cc.obj sdk\src\logs\CMakeFiles\opentelemetry_logs.dir\exporter.cc.obj sdk\src\logs\CMakeFiles\opentelemetry_logs.dir\event_logger_provider.cc.obj sdk\src\logs\CMakeFiles\opentelemetry_logs.dir\event_logger_provider_factory.cc.obj sdk\src\logs\CMakeFiles\opentelemetry_logs.dir\event_logger.cc.obj sdk\src\logs\CMakeFiles\opentelemetry_logs.dir\simple_log_record_processor.cc.obj sdk\src\logs\CMakeFiles\opentelemetry_logs.dir\simple_log_record_processor_factory.cc.obj sdk\src\logs\CMakeFiles\opentelemetry_logs.dir\batch_log_record_processor.cc.obj sdk\src\logs\CMakeFiles\opentelemetry_logs.dir\batch_log_record_processor_factory.cc.obj sdk\src\logs\CMakeFiles\opentelemetry_logs.dir\logger_context.cc.obj sdk\src\logs\CMakeFiles\opentelemetry_logs.dir\logger_context_factory.cc.obj sdk\src\logs\CMakeFiles\opentelemetry_logs.dir\multi_log_record_processor.cc.obj sdk\src\logs\CMakeFiles\opentelemetry_logs.dir\multi_log_record_processor_factory.cc.obj sdk\src\logs\CMakeFiles\opentelemetry_logs.dir\multi_recordable.cc.obj sdk\src\logs\CMakeFiles\opentelemetry_logs.dir\read_write_log_record.cc.obj sdk\src\logs\CMakeFiles\opentelemetry_logs.dir\readable_log_record.cc.obj  && cd ."
[85/147] cmd.exe /C "cd . && C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\x64\lib.exe  /machine:x64 /nologo /out:sdk\src\metrics\opentelemetry_metrics.lib sdk\src\metrics\CMakeFiles\opentelemetry_metrics.dir\async_instruments.cc.obj sdk\src\metrics\CMakeFiles\opentelemetry_metrics.dir\meter_provider.cc.obj sdk\src\metrics\CMakeFiles\opentelemetry_metrics.dir\meter_provider_factory.cc.obj sdk\src\metrics\CMakeFiles\opentelemetry_metrics.dir\meter.cc.obj sdk\src\metrics\CMakeFiles\opentelemetry_metrics.dir\meter_context.cc.obj sdk\src\metrics\CMakeFiles\opentelemetry_metrics.dir\meter_context_factory.cc.obj sdk\src\metrics\CMakeFiles\opentelemetry_metrics.dir\metric_reader.cc.obj sdk\src\metrics\CMakeFiles\opentelemetry_metrics.dir\instrument_metadata_validator.cc.obj sdk\src\metrics\CMakeFiles\opentelemetry_metrics.dir\export\periodic_exporting_metric_reader.cc.obj sdk\src\metrics\CMakeFiles\opentelemetry_metrics.dir\export\periodic_exporting_metric_reader_factory.cc.obj sdk\src\metrics\CMakeFiles\opentelemetry_metrics.dir\state\metric_collector.cc.obj sdk\src\metrics\CMakeFiles\opentelemetry_metrics.dir\state\observable_registry.cc.obj sdk\src\metrics\CMakeFiles\opentelemetry_metrics.dir\state\sync_metric_storage.cc.obj sdk\src\metrics\CMakeFiles\opentelemetry_metrics.dir\state\temporal_metric_storage.cc.obj sdk\src\metrics\CMakeFiles\opentelemetry_metrics.dir\aggregation\base2_exponential_histogram_indexer.cc.obj sdk\src\metrics\CMakeFiles\opentelemetry_metrics.dir\aggregation\histogram_aggregation.cc.obj sdk\src\metrics\CMakeFiles\opentelemetry_metrics.dir\aggregation\lastvalue_aggregation.cc.obj sdk\src\metrics\CMakeFiles\opentelemetry_metrics.dir\aggregation\sum_aggregation.cc.obj sdk\src\metrics\CMakeFiles\opentelemetry_metrics.dir\data\circular_buffer.cc.obj sdk\src\metrics\CMakeFiles\opentelemetry_metrics.dir\exemplar\filter.cc.obj sdk\src\metrics\CMakeFiles\opentelemetry_metrics.dir\exemplar\reservoir.cc.obj sdk\src\metrics\CMakeFiles\opentelemetry_metrics.dir\view\instrument_selector_factory.cc.obj sdk\src\metrics\CMakeFiles\opentelemetry_metrics.dir\view\meter_selector_factory.cc.obj sdk\src\metrics\CMakeFiles\opentelemetry_metrics.dir\view\view_factory.cc.obj sdk\src\metrics\CMakeFiles\opentelemetry_metrics.dir\view\view_registry_factory.cc.obj sdk\src\metrics\CMakeFiles\opentelemetry_metrics.dir\sync_instruments.cc.obj  && cd ."
[86/147] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\x64\cl.exe   /TP -DABSL_CONSUME_DLL -DHAVE_ABSEIL -DNOMINMAX -DOPENTELEMETRY_ABI_VERSION_NO=1 -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\x64-uwp-dbg\generated\third_party\opentelemetry-proto -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\api\include -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\sdk\include -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\sdk -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\ext\include -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\exporters\memory\include -external:IC:\v\vcpkg3\vcpkg_installed\x64-uwp\include -external:W0 /DWIN32 /D_WINDOWS /D_UNICODE /DUNICODE /DWINAPI_FAMILY=WINAPI_FAMILY_APP /D__WRL_NO_DEFAULT_LIB__ /nologo /Z7 /MP /GS /Gd /Gm- /W3 /WX- /Zc:wchar_t /Zc:inline /Zc:forScope /fp:precise /Oy- /EHsc  /utf-8  /Zc:__cplusplus /D_DEBUG /MDd /Od /RTC1 /showIncludes /Foexporters\memory\CMakeFiles\opentelemetry_exporter_in_memory.dir\src\in_memory_span_exporter_factory.cc.obj /Fdexporters\memory\CMakeFiles\opentelemetry_exporter_in_memory.dir\opentelemetry_exporter_in_memory.pdb /FS -c C:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\exporters\memory\src\in_memory_span_exporter_factory.cc
[87/147] cmd.exe /C "cd . && C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\x64\lib.exe  /machine:x64 /nologo /out:exporters\memory\opentelemetry_exporter_in_memory.lib exporters\memory\CMakeFiles\opentelemetry_exporter_in_memory.dir\src\in_memory_span_exporter_factory.cc.obj  && cd ."
[88/147] cmd.exe /C "cd /D C:\v\vcpkg3\buildtrees\opentelemetry-cpp\x64-uwp-dbg\opentelemetry-proto-prefix\src && C:\v\downloads\tools\cmake-3.27.1-windows\cmake-3.27.1-windows-i386\bin\cmake.exe -P C:/v/vcpkg3/buildtrees/opentelemetry-cpp/x64-uwp-dbg/opentelemetry-proto-prefix/tmp/opentelemetry-proto-gitclone.cmake && C:\v\downloads\tools\cmake-3.27.1-windows\cmake-3.27.1-windows-i386\bin\cmake.exe -E touch C:/v/vcpkg3/buildtrees/opentelemetry-cpp/x64-uwp-dbg/opentelemetry-proto-prefix/src/opentelemetry-proto-stamp/opentelemetry-proto-download"
Cloning into 'opentelemetry-proto'...
HEAD is now at c4dfbc5 Add maturity JSON entry per package
[89/147] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\x64\cl.exe   /TP -DABSL_CONSUME_DLL -DHAVE_ABSEIL -DNOMINMAX -DOPENTELEMETRY_ABI_VERSION_NO=1 -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\x64-uwp-dbg\generated\third_party\opentelemetry-proto -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\api\include -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\sdk\include -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\sdk -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\ext\include -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\exporters\ostream\include -external:IC:\v\vcpkg3\vcpkg_installed\x64-uwp\include -external:W0 /DWIN32 /D_WINDOWS /D_UNICODE /DUNICODE /DWINAPI_FAMILY=WINAPI_FAMILY_APP /D__WRL_NO_DEFAULT_LIB__ /nologo /Z7 /MP /GS /Gd /Gm- /W3 /WX- /Zc:wchar_t /Zc:inline /Zc:forScope /fp:precise /Oy- /EHsc  /utf-8  /Zc:__cplusplus /D_DEBUG /MDd /Od /RTC1 /showIncludes /Foexporters\ostream\CMakeFiles\opentelemetry_exporter_ostream_logs.dir\src\log_record_exporter.cc.obj /Fdexporters\ostream\CMakeFiles\opentelemetry_exporter_ostream_logs.dir\opentelemetry_exporter_ostream_logs.pdb /FS -c C:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\exporters\ostream\src\log_record_exporter.cc
[90/147] cmd.exe /C "cd /D C:\v\vcpkg3\buildtrees\opentelemetry-cpp\x64-uwp-dbg\opentelemetry-proto-prefix\src\opentelemetry-proto && C:\v\downloads\tools\cmake-3.27.1-windows\cmake-3.27.1-windows-i386\bin\cmake.exe -E echo_append && C:\v\downloads\tools\cmake-3.27.1-windows\cmake-3.27.1-windows-i386\bin\cmake.exe -E touch C:/v/vcpkg3/buildtrees/opentelemetry-cpp/x64-uwp-dbg/opentelemetry-proto-prefix/src/opentelemetry-proto-stamp/opentelemetry-proto-update"
[91/147] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\x64\cl.exe   /TP -DABSL_CONSUME_DLL -DHAVE_ABSEIL -DNOMINMAX -DOPENTELEMETRY_ABI_VERSION_NO=1 -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\x64-uwp-dbg\generated\third_party\opentelemetry-proto -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\api\include -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\sdk\include -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\sdk -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\ext\include -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\exporters\ostream\include -external:IC:\v\vcpkg3\vcpkg_installed\x64-uwp\include -external:W0 /DWIN32 /D_WINDOWS /D_UNICODE /DUNICODE /DWINAPI_FAMILY=WINAPI_FAMILY_APP /D__WRL_NO_DEFAULT_LIB__ /nologo /Z7 /MP /GS /Gd /Gm- /W3 /WX- /Zc:wchar_t /Zc:inline /Zc:forScope /fp:precise /Oy- /EHsc  /utf-8  /Zc:__cplusplus /D_DEBUG /MDd /Od /RTC1 /showIncludes /Foexporters\ostream\CMakeFiles\opentelemetry_exporter_ostream_span.dir\src\span_exporter.cc.obj /Fdexporters\ostream\CMakeFiles\opentelemetry_exporter_ostream_span.dir\opentelemetry_exporter_ostream_span.pdb /FS -c C:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\exporters\ostream\src\span_exporter.cc
[92/147] cmd.exe /C "cd . && C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\x64\lib.exe  /machine:x64 /nologo /out:exporters\ostream\opentelemetry_exporter_ostream_logs.lib exporters\ostream\CMakeFiles\opentelemetry_exporter_ostream_logs.dir\src\log_record_exporter.cc.obj exporters\ostream\CMakeFiles\opentelemetry_exporter_ostream_logs.dir\src\log_record_exporter_factory.cc.obj  && cd ."
[93/147] cmd.exe /C "cd /D C:\v\vcpkg3\buildtrees\opentelemetry-cpp\x64-uwp-dbg && C:\v\downloads\tools\cmake-3.27.1-windows\cmake-3.27.1-windows-i386\bin\cmake.exe -E echo_append && C:\v\downloads\tools\cmake-3.27.1-windows\cmake-3.27.1-windows-i386\bin\cmake.exe -E touch C:/v/vcpkg3/buildtrees/opentelemetry-cpp/x64-uwp-dbg/opentelemetry-proto-prefix/src/opentelemetry-proto-stamp/opentelemetry-proto-patch"
[94/147] cmd.exe /C "cd . && C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\x64\lib.exe  /machine:x64 /nologo /out:exporters\ostream\opentelemetry_exporter_ostream_span.lib exporters\ostream\CMakeFiles\opentelemetry_exporter_ostream_span.dir\src\span_exporter.cc.obj exporters\ostream\CMakeFiles\opentelemetry_exporter_ostream_span.dir\src\span_exporter_factory.cc.obj  && cd ."
[95/147] cmd.exe /C "cd /D C:\v\vcpkg3\buildtrees\opentelemetry-cpp\x64-uwp-dbg\opentelemetry-proto-prefix\src\opentelemetry-proto-build && C:\v\downloads\tools\cmake-3.27.1-windows\cmake-3.27.1-windows-i386\bin\cmake.exe -E echo_append && C:\v\downloads\tools\cmake-3.27.1-windows\cmake-3.27.1-windows-i386\bin\cmake.exe -E touch C:/v/vcpkg3/buildtrees/opentelemetry-cpp/x64-uwp-dbg/opentelemetry-proto-prefix/src/opentelemetry-proto-stamp/opentelemetry-proto-configure"
[96/147] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\x64\cl.exe   /TP -DABSL_CONSUME_DLL -DHAVE_ABSEIL -DNOMINMAX -DOPENTELEMETRY_ABI_VERSION_NO=1 -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\x64-uwp-dbg\generated\third_party\opentelemetry-proto -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\api\include -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\sdk\include -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\sdk -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\ext\include -IC:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\exporters\ostream\include -external:IC:\v\vcpkg3\vcpkg_installed\x64-uwp\include -external:W0 /DWIN32 /D_WINDOWS /D_UNICODE /DUNICODE /DWINAPI_FAMILY=WINAPI_FAMILY_APP /D__WRL_NO_DEFAULT_LIB__ /nologo /Z7 /MP /GS /Gd /Gm- /W3 /WX- /Zc:wchar_t /Zc:inline /Zc:forScope /fp:precise /Oy- /EHsc  /utf-8  /Zc:__cplusplus /D_DEBUG /MDd /Od /RTC1 /showIncludes /Foexporters\ostream\CMakeFiles\opentelemetry_exporter_ostream_metrics.dir\src\metric_exporter.cc.obj /Fdexporters\ostream\CMakeFiles\opentelemetry_exporter_ostream_metrics.dir\opentelemetry_exporter_ostream_metrics.pdb /FS -c C:\v\vcpkg3\buildtrees\opentelemetry-cpp\src\v1.12.0-c900296c5a.clean\exporters\ostream\src\metric_exporter.cc
[97/147] cmd.exe /C "cd /D C:\v\vcpkg3\buildtrees\opentelemetry-cpp\x64-uwp-dbg\opentelemetry-proto-prefix\src\opentelemetry-proto-build && C:\v\downloads\tools\cmake-3.27.1-windows\cmake-3.27.1-windows-i386\bin\cmake.exe -E echo_append && C:\v\downloads\tools\cmake-3.27.1-windows\cmake-3.27.1-windows-i386\bin\cmake.exe -E touch C:/v/vcpkg3/buildtrees/opentelemetry-cpp/x64-uwp-dbg/opentelemetry-proto-prefix/src/opentelemetry-proto-stamp/opentelemetry-proto-build"
[98/147] cmd.exe /C "cd . && C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\x64\lib.exe  /machine:x64 /nologo /out:exporters\ostream\opentelemetry_exporter_ostream_metrics.lib exporters\ostream\CMakeFiles\opentelemetry_exporter_ostream_metrics.dir\src\metric_exporter.cc.obj exporters\ostream\CMakeFiles\opentelemetry_exporter_ostream_metrics.dir\src\metric_exporter_factory.cc.obj  && cd ."
[99/147] cmd.exe /C "cd /D C:\v\vcpkg3\buildtrees\opentelemetry-cpp\x64-uwp-dbg\opentelemetry-proto-prefix\src\opentelemetry-proto-build && C:\v\downloads\tools\cmake-3.27.1-windows\cmake-3.27.1-windows-i386\bin\cmake.exe -E echo_append && C:\v\downloads\tools\cmake-3.27.1-windows\cmake-3.27.1-windows-i386\bin\cmake.exe -E touch C:/v/vcpkg3/buildtrees/opentelemetry-cpp/x64-uwp-dbg/opentelemetry-proto-prefix/src/opentelemetry-proto-stamp/opentelemetry-proto-install"
[100/147] cmd.exe /C "cd /D C:\v\vcpkg3\buildtrees\opentelemetry-cpp\x64-uwp-dbg && C:\v\downloads\tools\cmake-3.27.1-windows\cmake-3.27.1-windows-i386\bin\cmake.exe -E make_directory C:/v/vcpkg3/buildtrees/opentelemetry-cpp/x64-uwp-dbg/CMakeFiles && C:\v\downloads\tools\cmake-3.27.1-windows\cmake-3.27.1-windows-i386\bin\cmake.exe -E touch C:/v/vcpkg3/buildtrees/opentelemetry-cpp/x64-uwp-dbg/CMakeFiles/opentelemetry-proto-complete && C:\v\downloads\tools\cmake-3.27.1-windows\cmake-3.27.1-windows-i386\bin\cmake.exe -E touch C:/v/vcpkg3/buildtrees/opentelemetry-cpp/x64-uwp-dbg/opentelemetry-proto-prefix/src/opentelemetry-proto-stamp/opentelemetry-proto-done"
[101/147] cmd.exe /C "cd /D C:\v\vcpkg3\buildtrees\opentelemetry-cpp\x64-uwp-dbg && C:\v\vcpkg3\vcpkg_installed\x64-windows\tools\protobuf\protoc.exe --proto_path=C:/v/vcpkg3/buildtrees/opentelemetry-cpp/x64-uwp-dbg/opentelemetry-proto-prefix/src/opentelemetry-proto --cpp_out=C:/v/vcpkg3/buildtrees/opentelemetry-cpp/x64-uwp-dbg/generated/third_party/opentelemetry-proto --grpc_out=generate_mock_code=true:C:/v/vcpkg3/buildtrees/opentelemetry-cpp/x64-uwp-dbg/generated/third_party/opentelemetry-proto --plugin=protoc-gen-grpc="gRPC_CPP_PLUGIN_EXECUTABLE-NOTFOUND" C:/v/vcpkg3/buildtrees/opentelemetry-cpp/x64-uwp-dbg/opentelemetry-proto-prefix/src/opentelemetry-proto/opentelemetry/proto/common/v1/common.proto C:/v/vcpkg3/buildtrees/opentelemetry-cpp/x64-uwp-dbg/opentelemetry-proto-prefix/src/opentelemetry-proto/opentelemetry/proto/resource/v1/resource.proto C:/v/vcpkg3/buildtrees/opentelemetry-cpp/x64-uwp-dbg/opentelemetry-proto-prefix/src/opentelemetry-proto/opentelemetry/proto/trace/v1/trace.proto C:/v/vcpkg3/buildtrees/opentelemetry-cpp/x64-uwp-dbg/opentelemetry-proto-prefix/src/opentelemetry-proto/opentelemetry/proto/logs/v1/logs.proto C:/v/vcpkg3/buildtrees/opentelemetry-cpp/x64-uwp-dbg/opentelemetry-proto-prefix/src/opentelemetry-proto/opentelemetry/proto/metrics/v1/metrics.proto C:/v/vcpkg3/buildtrees/opentelemetry-cpp/x64-uwp-dbg/opentelemetry-proto-prefix/src/opentelemetry-proto/opentelemetry/proto/collector/trace/v1/trace_service.proto C:/v/vcpkg3/buildtrees/opentelemetry-cpp/x64-uwp-dbg/opentelemetry-proto-prefix/src/opentelemetry-proto/opentelemetry/proto/collector/logs/v1/logs_service.proto C:/v/vcpkg3/buildtrees/opentelemetry-cpp/x64-uwp-dbg/opentelemetry-proto-prefix/src/opentelemetry-proto/opentelemetry/proto/collector/metrics/v1/metrics_service.proto"
FAILED: generated/third_party/opentelemetry-proto/opentelemetry/proto/common/v1/common.pb.h generated/third_party/opentelemetry-proto/opentelemetry/proto/common/v1/common.pb.cc generated/third_party/opentelemetry-proto/opentelemetry/proto/resource/v1/resource.pb.h generated/third_party/opentelemetry-proto/opentelemetry/proto/resource/v1/resource.pb.cc generated/third_party/opentelemetry-proto/opentelemetry/proto/trace/v1/trace.pb.h generated/third_party/opentelemetry-proto/opentelemetry/proto/trace/v1/trace.pb.cc generated/third_party/opentelemetry-proto/opentelemetry/proto/logs/v1/logs.pb.h generated/third_party/opentelemetry-proto/opentelemetry/proto/logs/v1/logs.pb.cc generated/third_party/opentelemetry-proto/opentelemetry/proto/metrics/v1/metrics.pb.h generated/third_party/opentelemetry-proto/opentelemetry/proto/metrics/v1/metrics.pb.cc generated/third_party/opentelemetry-proto/opentelemetry/proto/collector/trace/v1/trace_service.pb.h generated/third_party/opentelemetry-proto/opentelemetry/proto/collector/trace/v1/trace_service.pb.cc generated/third_party/opentelemetry-proto/opentelemetry/proto/collector/logs/v1/logs_service.pb.h generated/third_party/opentelemetry-proto/opentelemetry/proto/collector/logs/v1/logs_service.pb.cc generated/third_party/opentelemetry-proto/opentelemetry/proto/collector/metrics/v1/metrics_service.pb.h generated/third_party/opentelemetry-proto/opentelemetry/proto/collector/metrics/v1/metrics_service.pb.cc generated/third_party/opentelemetry-proto/opentelemetry/proto/collector/trace/v1/trace_service.grpc.pb.h generated/third_party/opentelemetry-proto/opentelemetry/proto/collector/trace/v1/trace_service.grpc.pb.cc generated/third_party/opentelemetry-proto/opentelemetry/proto/collector/logs/v1/logs_service.grpc.pb.h generated/third_party/opentelemetry-proto/opentelemetry/proto/collector/logs/v1/logs_service.grpc.pb.cc generated/third_party/opentelemetry-proto/opentelemetry/proto/collector/metrics/v1/metrics_service.grpc.pb.h generated/third_party/opentelemetry-proto/opentelemetry/proto/collector/metrics/v1/metrics_service.grpc.pb.cc C:/v/vcpkg3/buildtrees/opentelemetry-cpp/x64-uwp-dbg/generated/third_party/opentelemetry-proto/opentelemetry/proto/common/v1/common.pb.h C:/v/vcpkg3/buildtrees/opentelemetry-cpp/x64-uwp-dbg/generated/third_party/opentelemetry-proto/opentelemetry/proto/common/v1/common.pb.cc C:/v/vcpkg3/buildtrees/opentelemetry-cpp/x64-uwp-dbg/generated/third_party/opentelemetry-proto/opentelemetry/proto/resource/v1/resource.pb.h C:/v/vcpkg3/buildtrees/opentelemetry-cpp/x64-uwp-dbg/generated/third_party/opentelemetry-proto/opentelemetry/proto/resource/v1/resource.pb.cc C:/v/vcpkg3/buildtrees/opentelemetry-cpp/x64-uwp-dbg/generated/third_party/opentelemetry-proto/opentelemetry/proto/trace/v1/trace.pb.h C:/v/vcpkg3/buildtrees/opentelemetry-cpp/x64-uwp-dbg/generated/third_party/opentelemetry-proto/opentelemetry/proto/trace/v1/trace.pb.cc C:/v/vcpkg3/buildtrees/opentelemetry-cpp/x64-uwp-dbg/generated/third_party/opentelemetry-proto/opentelemetry/proto/logs/v1/logs.pb.h C:/v/vcpkg3/buildtrees/opentelemetry-cpp/x64-uwp-dbg/generated/third_party/opentelemetry-proto/opentelemetry/proto/logs/v1/logs.pb.cc C:/v/vcpkg3/buildtrees/opentelemetry-cpp/x64-uwp-dbg/generated/third_party/opentelemetry-proto/opentelemetry/proto/metrics/v1/metrics.pb.h C:/v/vcpkg3/buildtrees/opentelemetry-cpp/x64-uwp-dbg/generated/third_party/opentelemetry-proto/opentelemetry/proto/metrics/v1/metrics.pb.cc C:/v/vcpkg3/buildtrees/opentelemetry-cpp/x64-uwp-dbg/generated/third_party/opentelemetry-proto/opentelemetry/proto/collector/trace/v1/trace_service.pb.h C:/v/vcpkg3/buildtrees/opentelemetry-cpp/x64-uwp-dbg/generated/third_party/opentelemetry-proto/opentelemetry/proto/collector/trace/v1/trace_service.pb.cc C:/v/vcpkg3/buildtrees/opentelemetry-cpp/x64-uwp-dbg/generated/third_party/opentelemetry-proto/opentelemetry/proto/collector/logs/v1/logs_service.pb.h C:/v/vcpkg3/buildtrees/opentelemetry-cpp/x64-uwp-dbg/generated/third_party/opentelemetry-proto/opentelemetry/proto/collector/logs/v1/logs_service.pb.cc C:/v/vcpkg3/buildtrees/opentelemetry-cpp/x64-uwp-dbg/generated/third_party/opentelemetry-proto/opentelemetry/proto/collector/metrics/v1/metrics_service.pb.h C:/v/vcpkg3/buildtrees/opentelemetry-cpp/x64-uwp-dbg/generated/third_party/opentelemetry-proto/opentelemetry/proto/collector/metrics/v1/metrics_service.pb.cc C:/v/vcpkg3/buildtrees/opentelemetry-cpp/x64-uwp-dbg/generated/third_party/opentelemetry-proto/opentelemetry/proto/collector/trace/v1/trace_service.grpc.pb.h C:/v/vcpkg3/buildtrees/opentelemetry-cpp/x64-uwp-dbg/generated/third_party/opentelemetry-proto/opentelemetry/proto/collector/trace/v1/trace_service.grpc.pb.cc C:/v/vcpkg3/buildtrees/opentelemetry-cpp/x64-uwp-dbg/generated/third_party/opentelemetry-proto/opentelemetry/proto/collector/logs/v1/logs_service.grpc.pb.h C:/v/vcpkg3/buildtrees/opentelemetry-cpp/x64-uwp-dbg/generated/third_party/opentelemetry-proto/opentelemetry/proto/collector/logs/v1/logs_service.grpc.pb.cc C:/v/vcpkg3/buildtrees/opentelemetry-cpp/x64-uwp-dbg/generated/third_party/opentelemetry-proto/opentelemetry/proto/collector/metrics/v1/metrics_service.grpc.pb.h C:/v/vcpkg3/buildtrees/opentelemetry-cpp/x64-uwp-dbg/generated/third_party/opentelemetry-proto/opentelemetry/proto/collector/metrics/v1/metrics_service.grpc.pb.cc 
cmd.exe /C "cd /D C:\v\vcpkg3\buildtrees\opentelemetry-cpp\x64-uwp-dbg && C:\v\vcpkg3\vcpkg_installed\x64-windows\tools\protobuf\protoc.exe --proto_path=C:/v/vcpkg3/buildtrees/opentelemetry-cpp/x64-uwp-dbg/opentelemetry-proto-prefix/src/opentelemetry-proto --cpp_out=C:/v/vcpkg3/buildtrees/opentelemetry-cpp/x64-uwp-dbg/generated/third_party/opentelemetry-proto --grpc_out=generate_mock_code=true:C:/v/vcpkg3/buildtrees/opentelemetry-cpp/x64-uwp-dbg/generated/third_party/opentelemetry-proto --plugin=protoc-gen-grpc="gRPC_CPP_PLUGIN_EXECUTABLE-NOTFOUND" C:/v/vcpkg3/buildtrees/opentelemetry-cpp/x64-uwp-dbg/opentelemetry-proto-prefix/src/opentelemetry-proto/opentelemetry/proto/common/v1/common.proto C:/v/vcpkg3/buildtrees/opentelemetry-cpp/x64-uwp-dbg/opentelemetry-proto-prefix/src/opentelemetry-proto/opentelemetry/proto/resource/v1/resource.proto C:/v/vcpkg3/buildtrees/opentelemetry-cpp/x64-uwp-dbg/opentelemetry-proto-prefix/src/opentelemetry-proto/opentelemetry/proto/trace/v1/trace.proto C:/v/vcpkg3/buildtrees/opentelemetry-cpp/x64-uwp-dbg/opentelemetry-proto-prefix/src/opentelemetry-proto/opentelemetry/proto/logs/v1/logs.proto C:/v/vcpkg3/buildtrees/opentelemetry-cpp/x64-uwp-dbg/opentelemetry-proto-prefix/src/opentelemetry-proto/opentelemetry/proto/metrics/v1/metrics.proto C:/v/vcpkg3/buildtrees/opentelemetry-cpp/x64-uwp-dbg/opentelemetry-proto-prefix/src/opentelemetry-proto/opentelemetry/proto/collector/trace/v1/trace_service.proto C:/v/vcpkg3/buildtrees/opentelemetry-cpp/x64-uwp-dbg/opentelemetry-proto-prefix/src/opentelemetry-proto/opentelemetry/proto/collector/logs/v1/logs_service.proto C:/v/vcpkg3/buildtrees/opentelemetry-cpp/x64-uwp-dbg/opentelemetry-proto-prefix/src/opentelemetry-proto/opentelemetry/proto/collector/metrics/v1/metrics_service.proto"
--grpc_out: protoc-gen-grpc: The system cannot find the file specified.




ninja: build stopped: subcommand failed.
```
</details>**Additional context**

<details><summary>vcpkg.json</summary>

```
{
  "name": "2",
  "version": "1",
  "dependencies": [
    {
      "name": "opentelemetry-cpp",
      "default-features": false,
      "features": [
        "otlp"
      ]
    }
  ]
}

```
</details>

--- a/ports/opentelemetry-cpp/portfile.cmake
+++ b/ports/opentelemetry-cpp/portfile.cmake
@@ -29,12 +29,12 @@ vcpkg_check_features(OUT_FEATURE_OPTIONS FEATURE_OPTIONS
 )
 
 # opentelemetry-proto is a third party submodule and opentelemetry-cpp release did not pack it.
-if(WITH_OTLP)
-    set(OTEL_PROTO_VERSION "0.19.0")
+if(WITH_OTLP_GRPC)
+    set(OTEL_PROTO_VERSION "1.0.0")
     vcpkg_download_distfile(ARCHIVE
         URLS "https://github.com/open-telemetry/opentelemetry-proto/archive/v${OTEL_PROTO_VERSION}.tar.gz"
         FILENAME "opentelemetry-proto-${OTEL_PROTO_VERSION}.tar.gz"
-        SHA512 b6d47aaa90ff934eb24047757d5fdb8a5be62963a49b632460511155f09a725937fb7535cf34f738b81cc799600adbbc3809442aba584d760891c0a1f0ce8c03
+        SHA512 74de78304a91fe72cfcdbd87fcb19c0d6338c161d6624ce09eac0527b1b43b8a5d8790ae055e1d3d44319eaa070a506f47e740f888c91d724a0aef8b509688f0
     )
 
     vcpkg_extract_source_archive(src ARCHIVE "${ARCHIVE}")

--- a/ports/opentelemetry-cpp/vcpkg.json
+++ b/ports/opentelemetry-cpp/vcpkg.json
@@ -51,7 +51,6 @@
     },
     "otlp-grpc": {
       "description": "Whether to include the OTLP gRPC exporter in the SDK",
-      "supports": "!uwp",
       "dependencies": [
         "grpc",
         {

--- a/ports/opentelemetry-cpp/vcpkg.json
+++ b/ports/opentelemetry-cpp/vcpkg.json
@@ -2,6 +2,7 @@
   "$schema": "https://raw.githubusercontent.com/microsoft/vcpkg-tool/main/docs/vcpkg.schema.json",
   "name": "opentelemetry-cpp",
   "version-semver": "1.12.0",
+  "port-version": 1,
   "description": [
     "OpenTelemetry is a collection of tools, APIs, and SDKs.",
     "You use it to instrument, generate, collect, and export telemetry data (metrics, logs, and traces) for analysis in order to understand your software's performance and behavior."
@@ -50,6 +51,7 @@
     },
     "otlp-grpc": {
       "description": "Whether to include the OTLP gRPC exporter in the SDK",
+      "supports": "!uwp",
       "dependencies": [
         "grpc",
         {

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -6318,7 +6318,7 @@
     },
     "opentelemetry-cpp": {
       "baseline": "1.12.0",
-      "port-version": 0
+      "port-version": 1
     },
     "opentelemetry-fluentd": {
       "baseline": "2.0.0",

--- a/versions/o-/opentelemetry-cpp.json
+++ b/versions/o-/opentelemetry-cpp.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "13178b82919223387a0d8d8efffae396a95d8110",
+      "version-semver": "1.12.0",
+      "port-version": 1
+    },
+    {
       "git-tree": "5fd8cacd9fd3aaa30b0ad133f2de052ba3ff869f",
       "version-semver": "1.12.0",
       "port-version": 0

--- a/versions/o-/opentelemetry-cpp.json
+++ b/versions/o-/opentelemetry-cpp.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "13178b82919223387a0d8d8efffae396a95d8110",
+      "git-tree": "71dd31a63ee43f477bad710ccce04ffac2be5238",
       "version-semver": "1.12.0",
       "port-version": 1
     },

